packages/cli/src/commands/setup/auth/templates/dbAuth.auth.ts.template

```diff
@@ -20,7 +20,11 @@ import { db } from './db'
  * fields to the `select` object below once you've decided they are safe to be
  * seen if someone were to open the Web Inspector in their browser.
  */
-export const getCurrentUser = async (session: DbAuthSession<number>) => {
+export const getCurrentUser = async (session: Decoded) => {
```

Comment:
Think you're missing webauthn template. Shall I merge the webauthn strict mode PR into this branch?

Reply:
Yes please 🙂